app/models/visitor.rb

```diff
@@ -0,0 +1,18 @@
+class Visitor < ActiveRecord::Base
+  include Person
+
+  belongs_to :visit
+  validates :visit, :rank, presence: true
```

Comment:
Perhaps `index`? Though that seems somewhat… loaded. Another I was wondering about was `order`? I'm not *too* averse to `rank`, mind you, but I agree that it's somewhat stilted.